src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,46 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+  private Set<Integer> numbers;
+
+  @BeforeEach
+  void setUp() {
+    numbers = new HashSet<>();
+    numbers.add(1);
+    numbers.add(1);
+    numbers.add(2);
+    numbers.add(3);
+  }
+
+  @Test
+  @DisplayName("Set의 크기 확인 테스트")
+  void size() {
+    assertThat(numbers.size()).isEqualTo(3);
+  }
+
+  @ParameterizedTest
+  @ValueSource(ints = {1, 2, 3})
+  @DisplayName("Set의 1, 2, 3 값 존재 확인 테스트")
+  void contains(int number) {
+    assertThat(numbers.contains(number)).isTrue();
```

Comment:
```suggestion
    assertThat(numbers).contains(1, 2, 3);
```
- assertion을 활용하여 의도를 분명히 드러내는 건 어떨까요